packages/compose/src/hooks/use-resize-observer/test/index.native.js

```diff
@@ -13,36 +13,32 @@ const TestComponent = ( { onLayout } ) => {
 	const [ resizeObserver, sizes ] = useResizeObserver();
 
 	return (
-		<View sizes={ sizes } onLayout={ onLayout }>
+		<View testID="test-component" sizes={ sizes } onLayout={ onLayout }>
```

Comment:
I see this is a very common way to query for elements with React Native, however, it's not recommended as per https://testing-library.com/docs/queries/bytestid and https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-the-wrong-query and particularly https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change#whats-with-the-data-testid-query. 

I don't have strong feelings though, since I realize this might be the only way to query for this particular element. Just wanted to bring it up in case there is a more reasonable alternative.

Reply:
Thanks for sharing those references!

For this case, it's a bit tricky since it's just a wrapper that has the `onLayout` callback and we need to manually trigger this event from the test. I was checking [this reference](https://callstack.github.io/react-native-testing-library/docs/how-should-i-query/#priority), so I feel like matching by ID is ok.

But definitely something to keep in mind for future tests 👍 

Thanks for reviewing! 🚀